server/types/oid/iterate.go

```diff
@@ -159,6 +179,51 @@ func IterateCurrentDatabase(ctx *sql.Context, callbacks Callbacks) error {
 	return nil
 }
 
+// iterateTypes iterates over all types that are present in the pg_type table.
+func iterateTypes(ctx *sql.Context, callbacks Callbacks) error {
+	allTypes := pgtypes.GetAllTypes()
+	// Exclude all types that are not present in the pg_type table
+	for _, t := range allTypes {
```

Comment:
We should use the index here for `ItemType.Index`. `GetAllTypes` is guaranteed to return the types in a deterministic order, so we don't need to do anything additional here.